.travis.yml

```diff
@@ -67,7 +67,7 @@ before_script:
   - if [ "$REDIS_ENABLE_IGBINARY" = true ] ; then pecl install igbinary ; fi
   - if [ "$REDIS_ENABLE_LZF" = true ] ; then printf "no\n" | pecl install lzf ; fi
   # Prepare system
-  - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
+  - if [ "$TEST_CONFIG" != "" ] ; then travis_retry ./bin/.travis/prepare_unittest.sh ; fi
```

Comment:
Are you sure this is going to work? [It depends on script return code](https://docs.travis-ci.com/user/common-build-problems/#travis_retry), and `prepare_unittest.sh` will always return 0, even if there are errors in subcommands. Error code from subcommand is not propagated.

Reply:
Good catch, I forgot about the "exit code is the code of last executed command" bit 😞 

As I see it, there are two options here:
1) use `set -e` in prepare_unittest.sh
2) check the return value of each composer command and exit with it if it's non-0.

WDYT?

Reply:
1

Reply:
BUT, several things in that script should ideally not be done several times. So this might cause some super strange issues down the line...